recipes/suitcase-csv/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "suitcase-csv" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "2f8eece9502084dcd81143e8ff95ad15ead43325f2fc063f0642789ffecd8a8f" %}
```

Comment:
@kbeeperez, following @ocefpaf's suggestion above, do you mind updating this place and the rest of the recipes in this PR to have the sha256 in the `source` block? Thanks!